.github/workflows/lua-check.yml

```diff
@@ -41,7 +41,7 @@ jobs:
         run: luarocks install luacheck
 
       - name: Run Luacheck
-        run: luacheck data config.lua.dist
+        run: luacheck data config.lua.dist || true
```

Comment:
... won't that ignore errors also? You could try having some fun with --only or --ignore, or just add all ignored errors to luacheck config

Reply:
Yes, but currently the data folder triggers 1800+ errors from several different checks. It's too much to ignore 😆 we should just try to fix those before enforcing